### PR TITLE
data: tuhamworld

### PR DESCRIPTION
**Pull request by tuhamworld**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/tuhamworld.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/tuhamworld",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://tuhamworld.hashnode.dev",
+      "icon": "SiHashnode"
+    },
```
Comment (sital002) [inline data/tuhamworld.json:20]:
```suggestion
    }
```
The trailing comma isnot required

File: data/tuhamworld.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Tunde Sanusi",
+  "bio": "I am a Frontend Engineer",
+  "avatar": "https://github.com/tuhamworld.png",
```
Comment (sital002) [inline data/tuhamworld.json:4]:
```suggestion
```
The avatar property isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉

**Pull request merged**